guides/concurrency.md

```diff
@@ -63,7 +63,7 @@ To spawn a fiber you use `spawn` with a block:
 ```crystal
 spawn do
   # ...
-  socket.gets
```

Comment:
The purpose of `socket.gets` is to show an abstract example of something you would *actually* do in another fiber; reading data from a socket (any socket). It isn't meant to be a compile-able example, and actually declaring a specific socket type and illustrating that is out of scope of the discussion. If desired, we could preface this example that explains that `socket` is abstract.

Moreover, reading input from `STDIN` is *not* typically something you would do in another fiber; for some kind of CLI application this is almost always going to be the "main" fiber of the application, so I don't think top level `gets` makes sense here.